pkg/kapp/resources/mod_field_copy.go

```diff
@@ -27,29 +28,36 @@ func (t FieldCopyMod) ApplyFromMultiple(res Resource, srcs map[FieldCopyModSourc
 		return nil
 	}
 
-	// Make a copy of resource, to avoid modifications
-	// that may be done even in case when there is nothing to copy
-	updatedRes := res.DeepCopy()
-
-	updated, err := t.apply(updatedRes.unstructured().Object, t.Path, Path{}, srcs)
-	if err != nil {
-		return fmt.Errorf("FieldCopyMod for path '%s' on resource '%s': %s", t.Path.AsString(), res.Description(), err)
-	}
-
-	if updated {
-		res.setUnstructured(updatedRes.unstructured())
+	for _, src := range t.Sources {
+		// Make a copy of resource, to avoid modifications
+		// that may be done even in case when there is nothing to copy
+		updatedRes := res.DeepCopy()
```

Comment:
Can we move it above line 39 from performance persepective.

Reply:
We have thought about this but as we are iterating on the sources like `[new,existing]` provided in Config so then whatever update happened for `new` resource will stay and can reflect when we iterate for `existing`

Reply:
Adopted.